calculateFrequency.js

```diff
@@ -0,0 +1,17 @@
+// Problem 2: Complete the calculateFrequency function that takes lowercase string as input and returns frequency of all english alphabet. (using only array, no in-built function)
+function calculateFrequency(string) {
+  var length;
+
+  length = string.length;
```

Comment:
If you are assigning the value in the very next line, its better to initialise and assign in a single line only.